antarest/launcher/repository.py

```diff
@@ -40,3 +40,13 @@ def delete(self, id: str) -> None:
         g = db.session.query(JobResult).get(id)
         db.session.delete(g)
         db.session.commit()
+
+    def delete_by_study_id(self, study_id: str) -> None:
+        job_results_to_delete: List[JobResult] = (
+            db.session.query(JobResult)
+            .filter(JobResult.study_id == study_id)
+            .all()
+        )
+        for job_result in job_results_to_delete:
+            db.session.delete(job_result)
```

Comment:
make this one query (replace all by delete) : https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.delete